FinanceServicesApi/V1/Boundary/Responses/PropertySummary/AssetAppointmentResponse.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+
+namespace FinanceServicesApi.V1.Boundary.Responses.PropertySummary
+{
+    public class AssetAppointmentResponse
```

Comment:
Its not appointment it should be apportionment, please make necessary changes wherever applicable.

Reply:
Renamed